### DISCUSSION TITLE
Support InfluxDB

### DISCUSSION THREAD

**Discussion by rusackas** [apache/superset#24027] [Ideas]
Starting this thread to capture interest, and/or seek volunteers who might be interested in working on this. There have been several requests for this, so I thought it might be worth consolidating things here:

https://github.com/apache/superset/discussions/18381
https://github.com/apache/superset/issues/20721
https://github.com/apache/superset/issues/16986
https://github.com/apache/superset/issues/5588

**Comment by suyashcjoshi** [comment]
Hello,

I've been working on my own implementation for the integration and can share here.

Thanks,
Suyash
